packages/wallets/src/features/cashier/modules/WithdrawalLocked/WithdrawalLocked.tsx

```diff
@@ -45,7 +45,7 @@ const WithdrawalLocked: React.FC<React.PropsWithChildren> = ({ children }) => {
         typeof minimumWithdrawal !== 'undefined' &&
         +remainder < minimumWithdrawal
     );
-    const isLoading = isCurrencyConfigLoading || isCryptoConfigLoading || !accountStatus;
+    const isLoading = isCurrencyConfigLoading || !accountStatus;
```

Comment:
Loading issue on load of withdrawal crypto is caused by this `isCryptoConfigLoading` added. Removed this check from WithdrawalLocked and moved it to the module instead